opentracing-impl-java8/src/main/java/io/opentracing/AbstractSpan.java

```diff
@@ -127,18 +123,46 @@ public String getBaggageItem(String key) {
     }
 
     @Override
-    public final Span log(String message, /* @Nullable */ Object payload) {
+    public final Span log(String event) {
+        return log(System.nanoTime() / 1000, event);
```

Comment:
Thanks - it just felt wrong leaving that in the code that could be considered "an example"